### ISSUE TITLE
[BUG] Pressing U on the botany pot can't view the recipes

### ISSUE THREAD

**Issue by Y-Xiao233** [Darkhax-Minecraft/BotanyPots#386]
### Minecraft Version

1.19.2

### Mod Version

BotanyPots-Forge-1.19.2-9.0.38 & 9.0.43

### Mod Loader

Forge

### What environment are you running the mod in?

All

### Issue Description

Press U against the botany pot in the backpack/rei, and the recipes cannot be seen in the rei.
But if you press R on the output of the existing recipe in the botany pot, you can see the recipe.
So I think it seems like a bug.
I'm not sure if you can fix this bug or add this feature to this mod. Thank you very much!

**Comment by Darkhax** [comment]
Hello, Botany Pots for 1.19.2 only has support for JEI so I can't guarantee everything will work as intended with REI. With that said, I just pushed an update to the JEI support that may help you on REI. There was previously a [JEI Bug](https://github.com/mezz/JustEnoughItems/issues/3183) that prevented me from registering the pots as recipe catalysts but that bug has been fixed on newer JEI versions. Hopefully this will also fix your issue as 1.19.2 is no longer receiving feature updates. 

https://www.curseforge.com/minecraft/mc-mods/botany-pots/files/5264073
